server/sonar-server/src/main/java/org/sonar/server/computation/step/ParseReportStep.java

```diff
@@ -69,7 +77,7 @@ public void execute(ComputationContext context) {
 
       // and parse!
       int rootComponentRef = reportMetadata.getRootComponentRef();
-      recursivelyProcessComponent(reader, context, rootComponentRef);
+      recursivelyProcessComponent(reader, context, rootComponentRef, defaultAssignee(context.getProjectSettings().getString(DEFAULT_ISSUE_ASSIGNEE)));
```

Comment:
Moreover calls to default assignee must not be frequent. And even if they are, I don't expect a big difference between calls to cached variable versus context.getProjectSettings().getString(DEFAULT_ISSUE_ASSIGNEE).
Cache over cache...